Example/Example/EditorDemoController.swift

```diff
@@ -108,28 +117,18 @@ class EditorDemoController: UIViewController {
         }
     }
 
-
     fileprivate var currentSelectedAttachment: MediaAttachment?
 
     var loadSampleHTML = false
 
-    private var shortcodePreProcessors = [Processor]()
-    private var shortcodePostProcessors = [Processor]()
-
     func setHTML(_ html: String) {
-        var processedHTML = html
-        for shortcodeProcessor in shortcodePreProcessors {
-            processedHTML = shortcodeProcessor.process(text: processedHTML)
-        }
-        richTextView.setHTML(processedHTML)
+        richTextView.setHTML(html)
     }
 
     func getHTML() -> String {
-        var processedHTML = richTextView.getHTML(prettyPrint: true)
-        for shortcodeProcessor in shortcodePostProcessors {
-            processedHTML = shortcodeProcessor.process(text: processedHTML)
-        }
-        return processedHTML
+        let html = richTextView.getHTML(prettyPrint: true)
+
+        return html
```

Comment:
Can we nuke the `html` var and return the result of `richTextView.getHTML(...)` directly?